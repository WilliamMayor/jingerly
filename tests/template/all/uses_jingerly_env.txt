This file contains {{ from_env }}