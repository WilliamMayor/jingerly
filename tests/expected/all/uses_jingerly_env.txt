This file contains a variable set in jingerly.env